dhis-2/dhis-api/src/main/java/org/hisp/dhis/dashboard/Dashboard.java

```diff
@@ -52,6 +54,16 @@
 
     private List<DashboardItem> items = new ArrayList<>();
 
+    /**
+     * Whether we restrict the filter dimensions the user may select.
+     */
+    private Boolean filterDimensionsAreRestricted = false;
+
+    /**
+     * Allowed filter dimensions (if any) the user may select.
+     */
+    private Set<String> allowedFilterDimensions = new HashSet<>();
```

Comment:
This should likely be a List instead of a Set. From the UI we will have a "selected" list for configuration and have the filters on the top bar of the dashboard, in which the order is of significance.